documentation/modules/storage-support.adoc

```diff
@@ -77,13 +77,31 @@ See link:https://access.redhat.com/documentation/en-us/openshift_container_platf
 If your migration uses block storage and persistent volumes created with an EXT4 file system, increase the file system overhead in CDI to be more than 10%. The default overhead that is assumed by CDI does not completely include the reserved place for the root partition. If you do not increase the file system overhead in CDI by this amount, your migration might fail.
 ====
 
+?? ANDY: The comments are for later when I break this note into two, one for OpenStack and for RHV
+// ifdef::ostack[]
 [NOTE]
 ====
-When migrating from OpenStack or running a cold-migration from RHV to the OCP cluster that MTV is deployed on, the migration allocates persistent volumes without CDI. In these cases, you might need to adjust the file system overhead.
+When you migrate from OpenStack, or when you run a cold migration from {rhv-full} to the OCP cluster that {project-short} is deployed on, the migration allocates persistent volumes without CDI. In these cases, you might need to adjust the file system overhead.
```

Comment:
I am assuming CDI is spelled out at the start.

Reply:
@shdeshpa07 I debated spelling it out, but if you think it's a good idea, then I will. Changed in the PR about prerequisites.